app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml

```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@color/ic_launcher_background"/>
-    <foreground android:drawable="@drawable/ic_launcher_foreground"/>
-</adaptive-icon>
```

Comment:
You seem to have deleted all the icons for the app. Note that these icons are required for compatibility with the old versions of Android. Revert all of them back again.

Reply:
Sorry for the mess.